src/mantine-rte/src/components/RichTextEditor/RichTextEditor.tsx

```diff
@@ -45,10 +45,13 @@ export interface RichTextEditorProps
   extends DefaultProps<RichTextEditorStylesNames>,
     Omit<React.ComponentPropsWithoutRef<'div'>, 'onChange'> {
   /** HTML content, value not forced as quill works in uncontrolled mode */
-  value: string | Delta;
+  value?: string | Delta;
+
+  /** Initial value of input */
+  defaultValue?: string | Delta;
```

Comment:
This will not work because the component does not pass thought defaultValue to Editor component. Either set defaultValue or remove this prop.

Reply:
Done! Thank you, I pass it